src/game_system.cpp

```diff
@@ -150,12 +150,12 @@ void Game_System::SePlay(const lcf::rpg::Sound& se, bool stop_sounds) {
 	// Validate
 	if (se.volume < 0 || se.volume > 100) {
 		Output::Debug("SE {} has invalid volume {}", se.name, se.volume);
-		volume = Utils::Clamp(se.volume, 0, 100);
+		volume = Utils::Clamp<uint32_t>(se.volume, 0, 100);
```

Comment:
Guess this should be `int32_t` for consistency.

Reply:
yeah indeed. ups ^^